Complete the `Changelog`, `Choke`, and `Combo fire` stubs (#6036)
ppy/osu-wiki: wiki/Glossary/Changelog/en.md

---
tags:
  - history
---

# Changelog

<!-- for the purpose of learning the history through the Wayback Machine (https://web.archive.org/), the changelog is available at different URLs:
  - http://osu.ppy.sh/?p=changelog
  - http://osu.ppy.sh/p/changelog
--->

> The [changelog](https://osu.ppy.sh/home/changelog) is a page where peppy and the osu!dev team detail their day-by-day improvements, additions, and bug fixes to the game. All changes from prior releases are recorded here for future browsers to view.

Reviewer: ```suggestion
The **[changelog](https://osu.ppy.sh/home/changelog)** is a page where the osu! development team details their day-by-day improvements, additions, and bug fixes to the game. All changes from prior releases are recorded here for archiving purposes.
```

> 

Reviewer: we can add something like that too, i guess?

```suggestion

The changelogs for the website and osu!(lazer) are generated automatically based on GitHub releases of those projects, while the rest are maintained manually.
```

Author: absolutely

## History

<!-- TODO(TicClick): wrap the images in infoboxes -->

> ![](img/changelog-web-2007.png "Web changelog (2007)")

Reviewer: ```suggestion
![](img/changelog-web-2007.png "The web changelog in 2007")
```

> The changelog has been started by peppy on 11th of September, 2007 in a dedicated forum thread titled "[Official Development Changelog](https://osu.ppy.sh/community/forums/topics/15)", where he would list notable changes and bug fixes, and sometimes share insights on upcoming plans.

Reviewer: ```suggestion
The changelog has been started by peppy on September 11, 2007 in a dedicated forum thread titled "[Official Development Changelog](https://osu.ppy.sh/community/forums/topics/15)", where he would list notable changes and bug fixes, and sometimes share insights on upcoming plans.
```

![](img/changelog-osume-2013.png "Changelog in the osu! updater (2013)")

In October, 2007, a web version of changelog [became available](https://osu.ppy.sh/community/forums/posts/2499), both through the website and the osu! updater.

> On 25th of October, 2009, the changelog [became available via the RSS](https://osu.ppy.sh/community/forums/topics/19137) feed.

Reviewer: ```suggestion
On October 25, 2009, the changelog [became available via the RSS](https://osu.ppy.sh/community/forums/topics/19137) feed.
```

![](img/build-propagation-graph-2015.png "Distribution of osu! build versions (2015)")

> On 28th of October, 2015, a graph with osu! versions distribution across the playerbase [was added to the changelog](https://web.archive.org/web/20151103161516/http://osu.ppy.sh:80/p/changelog), making the statistics open to the general public. At the same time, it became possible to filter changelog by update stream.

Reviewer: ```suggestion
On October 28, 2015, a graph with osu! versions distribution across the playerbase [was added to the changelog](https://web.archive.org/web/20151103161516/http://osu.ppy.sh:80/p/changelog), making the statistics open to the general public. At the same time, it became possible to filter changelog by update stream.
```

---
tags:
  - history
---

# Changelog

<!-- for the purpose of learning the history through the Wayback Machine (https://web.archive.org/), the changelog is available at different URLs:
  - http://osu.ppy.sh/?p=changelog
  - http://osu.ppy.sh/p/changelog
--->

> The **[changelog](https://osu.ppy.sh/home/changelog)** is a page where [the osu! development team](/wiki/People/The_Team/Developers) details their day-by-day improvements, additions, and bug fixes to the game. All changes from prior releases are recorded here for archiving purposes.

Reviewer: ```suggestion
The **[changelog](https://osu.ppy.sh/home/changelog)** is a page where the [osu! development team](/wiki/People/The_Team/Developers) details their day-by-day improvements, additions, and bug fixes to the game. All changes from prior releases are recorded here for archiving purposes.
```

The changelogs for the website and osu!(lazer) are generated automatically based on GitHub releases of those projects, while the rest are maintained manually.
## History

<!-- TODO(TicClick): wrap the images in infoboxes -->

![](img/changelog-web-2007.png "The web changelog in 2007")

The changelog has been started by peppy on September 11, 2007 in a dedicated forum thread titled "[Official Development Changelog](https://osu.ppy.sh/community/forums/topics/15)", where he would list notable changes and bug fixes, and sometimes share insights on upcoming plans.

![](img/changelog-osume-2013.png "Changelog in the osu! updater (2013)")

> In October, 2007, a web version of changelog [became available](https://osu.ppy.sh/community/forums/posts/2499), both through the website and the osu! updater.

Reviewer 2: ```suggestion
In October, 2007, a web version of changelog [became available](https://osu.ppy.sh/community/forums/posts/2499) both through the website and the osu! updater.
```

> On October 25, 2009, the changelog [became available via the RSS](https://osu.ppy.sh/community/forums/topics/19137) feed.

Reviewer 2: ```suggestion
On October 25, 2009, the changelog [became available via an RSS feed](https://osu.ppy.sh/community/forums/topics/19137).
```

![](img/build-propagation-graph-2015.png "Distribution of osu! build versions (2015)")

> On October 28, 2015, a graph with osu! versions distribution across the playerbase [was added to the changelog](https://web.archive.org/web/20151103161516/http://osu.ppy.sh:80/p/changelog), making the statistics open to the general public. At the same time, it became possible to filter changelog by update stream.

Reviewer 2: ```suggestion
On October 28, 2015, a graph with osu! versions distribution across the playerbase [was added to the changelog](https://web.archive.org/web/20151103161516/http://osu.ppy.sh:80/p/changelog), making the statistics open to the general public. At the same time, it became possible to filter the changelog by update stream.
```
